cmd/eudico-stats/README.md

```diff
@@ -23,5 +23,19 @@ cd cmd/eudico-stats && docker-compose up -d && cd -
 # step 5. launch eudico stats
 ./eudico-stats run --no-sync true
 
+# step 6. Add subnet
+./eudico wallet set-default <Your Key>
+./eudico subnet add --name test1 --consensus POW
+./eudico subnet join --subnet /root/t01000 4
 # you should be able to see the stats on http://localhost:9090 with eudico*
 ```
+
+Using docker-compose
+In the root directory
+```shell
+docker build -t eudico:latest -f Dockerfile.eudico .
```

Comment:
Nice!!! Have you tried to spawn a network with more than one node with Docker compose?